src/main/java/payapal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
+
+public class Perfection {
+
+    private static Perfection perf;
+
+    public synchronized static Perfection getPerf() {
```

Comment:
Since you are using a single design pattern you should rename this method to getInstance().
More info: https://en.wikipedia.org/wiki/Singleton_pattern